src/tools/web/client/src/components/CreateKey.jsx

```diff
@@ -20,6 +20,11 @@ export default class CreateKey extends React.Component {
   render () {
     const { setKey } = this.props
 
+    // remove / prefix/suffix
```

Comment:
slashes are escaped on a much lower level (in the `kdb.js` "library"), so this shouldn't cause an issue. it is only cleaning user input from unnecessary `/` prefixes/suffixes